src/main/java/carRacing/view/InputView.java

```diff
@@ -0,0 +1,43 @@
+package carRacing.view;
+
+import java.util.Scanner;
+
+public class InputView {
+
+	private static final int MINIMUM_NATURAL_NUM = 1;
+	private static final String REGEX = ",";
+
+	private Scanner sc = new Scanner(System.in);
+
+	public int inputMoveCount() {
+		return getValidCount("시도할 회수는 몇 회 인가요?");
+	}
+
+	public int getValidCount(String text) {
+		System.out.println(text);
+		int numberOfAttempts = sc.nextInt();
+		if (numberOfAttempts < MINIMUM_NATURAL_NUM) {
+			throw new IllegalArgumentException("1 이상의 자연수를 입력해주세요.");
+		}
+		return numberOfAttempts;
+	}
+
+	public String[] inputCarName() {
+		return getNames("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분)");
+	}
+
+	private String[] getNames(String text) {
+		System.out.println(text);
```

Comment:
```suggestion
	public String[] inputCarName() {
		return getNames();
	}

	private String[] getNames() {
		System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분)");
		String[] carNames = sc.nextLine().split(REGEX);
```
파라미터를 getNames에서 활용하고 있지 않으므로 넘길필요 없이 바로 사용해도 될 것 같아요